_pages/inclusiveness-diversity-accessibility.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: single
+title: Inclusiveness, diversity and accessibility
+tagline: Azure User Group Belgium
+permalink: /inclusiveness-diversity-accessibility
+---
+
+At AZUG, we care deeply about making our events a safe and welcoming space for everyone involved.
+
+Given AZUG is driven by volunteers, this does involve making trade-offs due to available time and effort. We want to prioritize those efforts that have the most impact on safety, comfort, diversity, and accessibility.
+
+On this page, we will list the things we are doing, those that we consider and those that we decided not to do for the time being.
+
+Do provide us your feedback on these - if you are affected by these efforts, if you have relevant experiences, or if you are simply a bystander who cares, let us know what you would like to see us do differently.
+
+Please send your ideas or concerns to [getintouchwithus@azug.be](getintouchwithus@azug.be). We treat all submissions confidentially.
```

Comment:
we treat all submissions confidentially and equally.

Reply:
Thanks, fixed.